doc/sphinx/conf.py

```diff
@@ -146,7 +146,8 @@ def whereis(binary):
     "use_repository_button": True,
     "use_download_button": True,
     "path_to_docs": "doc",
-    "use_issues_button": True
+    "use_issues_button": True,
+    "extra_footer": "<div>oneDNN is licensed under Apache License Version 2.0. Refer to the <a href='https://github.com/uxlfoundation/oneDNN/blob/main/LICENSE'>LICENSE</a> file for the full license text and copyright notice.</div>"
```

Comment:
`<div>` does not seem necessary here as the theme already wraps this text.

```suggestion
    "extra_footer": "oneDNN is licensed under Apache License Version 2.0. Refer to the <a href='https://github.com/uxlfoundation/oneDNN/blob/main/LICENSE'>LICENSE</a> file for the full license text and copyright notice."
```